LoopKit/InsulinKit/CachedInsulinDeliveryObject+CoreDataClass.swift

```diff
@@ -201,7 +201,7 @@ extension CachedInsulinDeliveryObject {
             value: programmedValue,
             unit: unit,
             deliveredUnits: !isMutable ? deliveredUnits : nil,
-            description: nil,
+            description: deletedAt == nil ? nil : "deletedAt: \(String(describing: deletedAt))",
```

Comment:
Would maybe prefer that we print out the CachedInsulinDeliveryObject objects themselves in the issue report, including the deletedAt, rather than shoehorning the deleted attribute into the description field.  That might have other benefits as we'd be displaying a closer version of what's actually stored, instead of transforming them into `DoseEntry` objects.  Maybe there are other omissions or information losses in that transformation.  But I won't block on it; not sure how much work to get a stringified version of CachedInsulinDeliveryObject is.